core/modules/ccontrol/UserQuery.h

```diff
@@ -33,108 +33,52 @@
 
 // System headers
 #include <memory>
-#include <mutex>
 
 // Third-party headers
```

Comment:
I don't know if there is any specific policy on this, I sometimes leave it there in case I need to add more includes in the future. I don't remember exactly what text I have to put in the comment if I have to add a section and it's annoying to have to search other files for an example.